lib/Sema/CSSimplify.cpp

```diff
@@ -10771,8 +10771,13 @@ bool ConstraintSystem::simplifyAppliedOverloadsImpl(
   // Don't attempt to filter overloads when solving for code completion
   // because presence of code completion token means that any call
   // could be malformed e.g. missing arguments e.g. `foo([.#^MEMBER^#`
-  if (isForCodeCompletion())
-    return false;
+  if (isForCodeCompletion()) {
+    bool ArgContainsCCTypeVar = Type(argFnType).findIf(
```

Comment:
I just realized that this could be simplified down to `Type(argFnType).findIf(isCodeCompletionTypeVar);`